erc20-bridge-token/test/BridgeToken.js

```diff
@@ -159,41 +180,51 @@ describe('BridgeToken', () => {
     lockResultProof.outcome_proof.outcome.receipt_ids[0] = 'C'.repeat(44);
     await BridgeTokenFactory.pause()
     await expect(
-      BridgeTokenFactory.deposit(borshifyOutcomeProof(metadataProof), 1090)
+      BridgeTokenFactory.deposit(borshifyOutcomeProof(lockResultProof), 1090)
     )
       .to
       .be
       .revertedWith('Pausable: paused');
   })
   it('withdraw token', async function () {
     const { tokenProxyAddress, token } = await createEmptyToken(nearTokenId, BridgeTokenFactory, BridgeTokenInstance)
-    const metadataProof = require('./proof_template.json');
-    metadataProof.outcome_proof.outcome.status.SuccessValue = serialize(SCHEMA, 'SetMetadataResult', createDefaultERC20Metadata(nearTokenId, 1089)).toString('base64');
-    await BridgeTokenFactory.setMetadata(borshifyOutcomeProof(metadataProof), 1089);
+    await setMetadata(nearTokenId, tokenProxyAddress);
 
-    const lockResultProof = metadataProof;
+    const amountToTransfer = 100;
+    const recipient = "testrecipient.near";
+    const lockResultProof = require('./proof_template.json');
     lockResultProof.outcome_proof.outcome.status.SuccessValue = serialize(SCHEMA, 'LockResult', {
       prefix: RESULT_PREFIX_LOCK,
       token: nearTokenId,
-      amount: 100,
+      amount: amountToTransfer,
       recipient: ethers.utils.arrayify(adminAccount.address),
     }).toString('base64');
     lockResultProof.outcome_proof.outcome.receipt_ids[0] = 'D'.repeat(44);
     await BridgeTokenFactory.setTokenWhitelistMode(nearTokenId, 2);
-    await BridgeTokenFactory.deposit(borshifyOutcomeProof(metadataProof), 1090);
+    await BridgeTokenFactory.deposit(borshifyOutcomeProof(lockResultProof), 1090);
```

Comment:
Please use a constant for block proof height here and in all other cases

Reply:
699afa2dcb032eb54349c0516950d007e028b2b1